source/docs/configuration.md

```diff
@@ -12,62 +19,72 @@ title: Configuring Embark
   "buildDir": "dist/",
   "config": "config/",
   "versions": {
-    "web3": "1.0.0-beta.34",
-    "solc": "0.4.17",
+    "web3": "1.0.0-beta",
+    "solc": "0.4.25",
     "ipfs-api": "17.2.4"
   },
   "plugins": {
   },
   "options": {
+    "solc": {
+      "optimize": true,
+      "optimize-runs": 200
+    }
   }
 }
 </code></pre>
 
+Let's look at the different options and learn what they do and mean.
+
 ### contracts
 
-List of directories in which embark should look for contracts. These typically are GLOB expressions (e.g `["contracts/**/*.sol"]` will match all sol files inside any folders inside `contracts/`)
+This is a list of directories in which Embark should look for Smart Contract files. These typically are globbing patterns (e.g `["contracts/**/*.sol"]` will match all sol files inside any folders inside `contracts/`).
+
+### app
 
-### pipeline
+Everything inside `app` configures the assets of our application. At the moment there are three types of assets to configure: JavaScript files, HTML and images.
```

Comment:
Not 100% true. It works for CSS files and all other assets that need to be simply copied.

Reply:
Keys under `"app"` that end in `.js` are basically output files for webpack, and the files in the arrays corresponding to those `.js` keys are each an entry point for webpack that will be packed (along with their respective dependencies) and then included together in the respective output file. Keys that don't end in `.js`, whether a directory or single file, are copy-targets for the file/s / glob-patterns corresponding to those keys — the contents aren't run through webpack, simply copied as @jrainville says.

Reply:
Okay, let me reword this.

Reply:
Updated!